include/fmt/compile.h

```diff
@@ -164,7 +164,7 @@ struct is_compiled_string : std::is_base_of<compiled_string, S> {};
 #endif
 
 #if FMT_USE_NONTYPE_TEMPLATE_PARAMETERS
-template <typename Char, size_t N, fixed_string<Char, N> Str>
+template <typename Char, size_t N, fmt::detail_exported::str<Char, N> Str>
```

Comment:
I suggest keeping the old name (fixed_string) which is more descriptive.

Reply:
Right.
My concern is the size of these `fmt::v7::detail_exported::fixed_string` linker symbols. This is just the common prefix. If you look at real code, the actual linker symbols with the specialized fixed strings become horribly long. My be I can convince Cameron to use a different mangling, may be derived from the base64-encoded SHA1 of the string value.

Anyway, I will change that.